src/components/Admin/ListSkills/ListSkills.js

```diff
@@ -247,6 +247,7 @@ class ListSkills extends React.Component {
           skillsData: skills,
           loading: false,
         });
+        console.log(skills);
```

Comment:
Please remove this console log if it is not required.

Reply:
Done. Please review. Thanks :-)